packages/gitbook/src/lib/references.tsx

```diff
@@ -35,6 +36,8 @@ export interface ResolvedContentRef {
     active: boolean;
     /** File, if the reference is a file */
     file?: RevisionFile;
+    /** ID of an attached document. */
+    documentId?: string;
```

Comment:
Is it not better to do:

```suggestion
    reusableContent?: RevisionReusableContent;
```

otherwise it's unclear which case will define a document